src/wrappers/httpwrapper.h

```diff
@@ -32,17 +40,37 @@ class Client
         return wrappedResult(res);
     }
 
-    Result post(std::string const &path, std::string const &message)
+    Result post(std::string const &path, std::string const &message, ContentType const &contentType = textPlain)
     {
-        char const *contentType = "text/plain";
-        auto const res = m_client.Post(path.c_str(), message, contentType);
+        auto const res = m_client.Post(path.c_str(), message, getContentType(contentType).c_str());
 
         return wrappedResult(res);
     }
 
 private:
+    std::string getContentType(ContentType const &contentType) const
+    {
+        switch (contentType)
+        {
+            case textPlain:
+            {
+                return CONTENT_TYPE_PLAIN_TEXT;
```

Comment:
Since this CPP SDK will be mostly used to interact with the Elrond APIs, which are 100% JSON-based, perhaps adjust the default values? (here and above)

Reply:
Changed default content type to JSON-based !